libminifi/test/aws-tests/PutS3ObjectTests.cpp

```diff
@@ -107,6 +107,17 @@ TEST_CASE_METHOD(PutS3ObjectTestsFixture, "Test required property not set", "[aw
   REQUIRE_THROWS_AS(test_controller.runSession(plan, true), minifi::Exception&);
 }
 
+TEST_CASE_METHOD(PutS3ObjectTestsFixture, "Test incomplete credentials in credentials service", "[awsS3Config]") {
+  setBucket();
+  plan->setProperty(aws_credentials_service, "Secret Key", "secret");
+  setCredentialsService();
+  REQUIRE_THROWS_AS(test_controller.runSession(plan, true), minifi::Exception&);
+  REQUIRE(verifyLogLinePresenceInPollTime(std::chrono::seconds(3), "AWS Credentials have not been set!"));
+
+  // Test that no invalid credentials file was set from previous properties
+  REQUIRE(!LogTestController::getInstance().contains("load configure file failed", std::chrono::seconds(0), std::chrono::milliseconds(0)));
```

Comment:
`REQUIRE_FALSE` might be used here

Reply:
Updated in 91ec4bf4a04a146344ceab44830f375ed6278338